File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/NoteResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class NoteResponse 
{
    public bool Valid { get; set; }
    public string Message { get; set; }
    public double VarVersion { get; set; }
    public bool SerializeNulls { get; set; }
    public long StartTimeLog { get; set; }
    public string ErrorCode { get; set; }
    public List<NameStringValueResponse> Request { get; set; }
    public long NoteId { get; set; }
    public string Comment { get; set; }
    public string Tags { get; set; }
    public long LikesCount { get; set; }
    public long DislikesCount { get; set; }
    public long DateCreated { get; set; }
    public long CreatedDate { get; set; }
    public long UpdatedDate { get; set; }
    public AccountShortResponse Owner { get; set; }
    public double Latitude { get; set; }
    public double Longitude { get; set; }
    public string LocationDescription { get; set; }
    public string Returning { get; set; }
}

File: server-scala-play-server/app/model/NoteResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for NoteResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class NoteResponse(
  valid: Option[Boolean],
  message: Option[String],
  version: Option[Double],
  serializeNulls: Option[Boolean],
  startTimeLog: Option[Long],
  errorCode: Option[String],
  request: Option[List[NameStringValueResponse]],
  noteId: Option[Long],
  comment: Option[String],
  tags: Option[String],
  likesCount: Option[Long],
  dislikesCount: Option[Long],
  dateCreated: Option[Long],
  createdDate: Option[Long],
  updatedDate: Option[Long],
  owner: Option[AccountShortResponse],
  latitude: Option[Double],
  longitude: Option[Double],
  locationDescription: Option[String],
  returning: Option[String]
)

object NoteResponse {
  implicit lazy val noteResponseJsonFormat: Format[NoteResponse] = Json.format[NoteResponse]
}

